nexus/src/internal_api/http_entrypoints.rs

```diff
@@ -771,6 +779,29 @@ async fn bgtask_view(
     apictx.internal_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Activates one or more background tasks, causing them to be run immediately
+/// if idle, or scheduled to run again as soon as possible if already running.
+#[endpoint {
+    method = POST,
+    // Can't be "bgtasks/activate" because it conflicts with the GET endpoint
+    // for bgtask_view.
+    path = "/bgtasks-activate",
```

Comment:
Awkward! I guess this is because as far as dropshot / OpenAPI is concerned, we could have a background task _named_ `activate`? Would it make sense to move the GET endpoints down a level? `/bgtasks/status/{bgtask_name}` or something like that

Reply:
It's dropshot that enforces that you can't have static paths and variable paths that overlap.  It obviously could be made to work, but seemed like the sort of thing we want to avoid.  I'd be fine with moving that down to `/bgtasks/status`.  It'll be a runtime flag day between `omdb` and Nexus but I don't imagine a lot of people use mismatched `omdb`s and invoke these commands a lot.

Reply:
Yeah. Completely agree that we want to avoid conflicts here -- I'll change the API to `/bgtasks/status` etc, and just assume that the omdb flag day isn't a huge deal.